app/utilities/schema.py

```diff
@@ -114,11 +117,15 @@ def load_schema_from_metadata(
         # This should not be used in production.
 
         start = time.time()
-        schema = load_schema_from_url(schema_url, language_code)
+        schema = load_schema_from_url(
+            schema_url=schema_url,
+            language_code=language_code,
+            parameters=ImmutableDict({"language": language_code}),
```

Comment:
This is odd as we are passing the same thing in two ways.

Given CIR doesn't support languages and schema url is not used for prod and likely to be removed, I suggest remove the language parameter from the url and completely from the `load_schema_from_url` until we need it, for CIR it won't be as simple as a query param anyway.

You can this pass guid as a `**kwargs`, maybe name it `**query_params`

Reply:
I agree its odd, good to remove it but just to confirm, you're suggesting I change the existing
`constructed_schema_url = f"{schema_url}?language={language}"` to `constructed_schema_url = f"{schema_url}` in the case of load schema from url - even if not used in production, won't that break it? or does that not matter. What was the intended use case for it?